### DISCUSSION TITLE
Welcome to airel-client Discussions!

### DISCUSSION THREAD

**Discussion by scottBowles** [scottBowles/airel-client#29] [Announcements]
## Welcome!
Feel free to start a discussion or respond to an ongoing one.

This is a good place to bring up possible features, think together about future design, or play around however you want.

If you have a bug to report, go over to the [Issues tab](https://github.com/scottBowles/airel-client/issues) and create an issue. There are some things over there that probably should be here (I just discovered Github Discussions). You're welcome to create issues and comment on anything over there too.

**Comment by GlitterSoap** [comment]
you are not alone. though i am wildly out if my element here

**Reply by scottBowles** [reply]
Welcome! I don't know about out of your element though. This part's basically a message board, and for that matter, you used to practically code up your phone
